index.html

```diff
@@ -1566,61 +1551,70 @@ <h4>CoAP Default Vocabulary Terms</h4>
-                <code>[ { "cov:optionName": "Observe", "cov:optionValue": 1 }]</code></td>
+              <td><code>"subprotocol":"cov:observe"</code></td>
             </tr>
             <tr>
               <td><code>invokeaction</code></td>
-              <td> <code>"cov:methodCode": 2 </code> OR <code>"cov:methodName": "POST"</code></td>
+              <td><code>"cov:methodName": "POST"</code></td>
             </tr>
             <tr>
               <td><code>subscribeevent</code></td>
-              <td> <code>"cov:methodCode": 1</code> OR <code>"cov:methodName": "GET"</code> with
-                <code>[ { "cov:optionName": "Observe", "cov:optionValue": 0 }]</code></td>
+              <td><code>"subprotocol":"cov:observe"</code></td>
             </tr>
             <tr>
               <td><code>unsubscribeevent</code></td>
-              <td> <code>"cov:methodCode": 1</code> OR <code>"cov:methodName": "GET"</code> with
-                <code>[ { "cov:optionName": "Observe", "cov:optionValue": 1 }]</code></td>
+              <td><code>"subprotocol":"cov:observe"</code></td>
+
             </tr>
-            <!--
             <tr>
               <td><code>readallproperties</code></td>
-              <td> <code>"cov:methodCode": 1</code></td>
+              <td> <code>"cov:methodName": "GET"</code></td>
             </tr>
             <tr>
               <td><code>writeallproperties</code></td>
-              <td> <code>"cov:methodCode": 3</code></td>
+              <td> <code>"cov:methodName": "PUT"</code></td>
             </tr>
             <tr>
               <td><code>readmultipleproperties</code></td>
-              <td> <code>"cov:methodCode": 1</code></td>
+              <td> <code>"cov:methodName": "GET"</code></td>
             </tr>
             <tr>
               <td><code>writemultipleproperties</code></td>
-              <td> <code>"cov:methodCode": 3</code></td>
-            </tr> -->
+              <td> <code>"cov:methodName": "PUT"</code></td>
+            </tr>
           </tbody>
         </table>
+        </section>
+
+        Observing Resources in CoAP should be done as specified in [[!RFC7641]].
+        Since observing and unobserving need to be indicated with the Observe flag in the header, this mechanism can be also described in the <code> cov:options </code> with the value <code>[ { "cov:optionName": "Observe", "cov:optionValue": 0 }]</code>. 
+        However, this is not enough to describe the mechanism of getting asynchronous updates from the observed resource.
+        Thus, <code>"subprotocol":"cov:observe"</code> is used to indicate the observation mechanism.
+
+        <p class="ednote">
+          As indicated by the <a href="#coap-default-vocabulary-terms" class="sec-ref">CoAP Default Vocabulary Terms</a>, it is recommended to use the <code>subprotocol</code> to describe the observation and using both the <code> cov:options </code> and <code>subprotocol</code> can lead to confusion.
```

Comment:
For that reason, I would probably say `MUST NOT` on `[ { "cov:optionName": "Observe", "cov:optionValue": 0 }]`.

Reply:
Addressed in https://github.com/w3c/wot-binding-templates/pull/82/commits/3ff9303538e65e12ef1b3e48f0e7961940d023af

Reply:
However, I am not sure if this is what you meant

Reply:
Right now (even with the update) it sounds like a consumer can freely choose between the subprotocol and `[ { "cov:optionName": "Observe", "cov:optionValue": 0 }]` (with the hard requirement not to have both and a soft preference for the former). IMO we should forbid the latter entirely, as I cannot imagine any scenario where it would be useful that a consumer sets the Observe option without understanding what it's doing. (That's how we defined `cov:options`, right?)